src/writer/settings.ts

```diff
@@ -3,7 +3,7 @@ import isEqual from 'lodash/isEqual';
 import { storeSettings } from '../helpers/adapters/mysql';
 import { jsonParse } from '../helpers/utils';
 import { spaces } from '../helpers/spaces';
-import { getSpaceUri } from '../helpers/ens';
+import { getSpaceUri } from '@snapshot-labs/snapshot.js/src/utils';
```

Comment:
This can be removed since we already import the entire lib in line 1. Usage needs to be adjusted further down.

```suggestion
```
and
```js
const spaceUri = await getSpaceUri(msg.space);
```
would need to be
```js
const spaceUri = await snapshot.utils.getSpaceUri(msg.space);
```